prototype/genetic_curve_fitting_prototype/genetic_curve_fitting.py

```diff
@@ -146,12 +149,13 @@ def main():
   for val in bestValues:
     equation += str(val*limits[exponent])
     equation += "x^" + str(exponent) + " "
-    exponent += 1     
+    exponent += 1 
+      
   print(equation)
 
   plt.plot(line_x, line_y)
   plt.show()
   port.close()
 
 if __name__ == "__main__":
-  main()
+  curveFitting()
```

Comment:
You've changed the name of the method, but I think you forgot to update the name of the method in the Readme.md file.